### PR TITLE
Fix promotion localvar6

### PR DESCRIPTION
**Pull request by cosunae**

### CODE REVIEW

**Comment by cosunae** [comment]
launch jenkins

**Comment by cosunae** [comment]
launch jenkins

**Comment by cosunae** [comment]
launch pipeline

File: bundle/CMakeLists.txt [hunk 1]
```diff
@@ -90,8 +90,8 @@ yoda_find_package(
   REQUIRED_VARS dawn_DIR
   ADDITIONAL
     DOWNLOAD_DIR ${YODA_DOWNLOAD_DIR}
-    GIT_REPOSITORY "git@github.com:MeteoSwiss-APN/dawn.git"
-    GIT_TAG "master" 
+    GIT_REPOSITORY "git@github.com:cosunae/dawn.git"
```
Comment (twicki) [inline bundle/CMakeLists.txt]: change the bundle back :)

**Comment by twicki** [comment]
launch pipeline

**Comment by twicki** [comment]
launch pipeline

**Comment by twicki** [comment]
launch pipeline

File: bundle/CMakeLists.txt (hunk 1 above)
Comment (cosunae) [inline bundle/CMakeLists.txt]:
```suggestion
    GIT_REPOSITORY "git@github.com:MeteoSwiss-APN/dawn.git"
```

File: bundle/CMakeLists.txt [hunk 2]
```diff
@@ -90,8 +90,8 @@ yoda_find_package(
   REQUIRED_VARS dawn_DIR
   ADDITIONAL
     DOWNLOAD_DIR ${YODA_DOWNLOAD_DIR}
-    GIT_REPOSITORY "git@github.com:MeteoSwiss-APN/dawn.git"
-    GIT_TAG "master" 
+    GIT_REPOSITORY "git@github.com:cosunae/dawn.git"
+    GIT_TAG "fix_promotion_localvar6"
```
Comment (cosunae) [inline bundle/CMakeLists.txt]:
```suggestion
    GIT_TAG "master" 
```

**Pull request merged**